src/components/display-rectangle.js

```diff
@@ -0,0 +1,28 @@
+const {DOM, createClass} = require("react");
+const {div, canvas} = DOM;
+const React = require("react");
+
+const DisplayRectangle = createClass({
+    render: function() {
```

Comment:
I thought we had agreed to all use `render() { ...` instead of `render: function() { ...`? (just a nitpick)